dll/win32/fltlib/fltlib.c

```diff
@@ -1,10 +1,10 @@
 /*
-* PROJECT:         Filesystem Filter Manager library
-* LICENSE:         GPL - See COPYING in the top level directory
-* FILE:            dll/win32/fltlib/fltlib.c
-* PURPOSE:
-* PROGRAMMERS:     Ged Murphy (ged.murphy@reactos.org)
-*/
+ * PROJECT:     ReactOS system libraries
```

Comment:
The previous project name was more descriptive?

Reply:
Previous project name has moved to "PURPOSE", any suggestion about "PROJECT" and "PURPOSE" here?

Reply:
In principle, "Project" is the overall description for (here) fltlib, while "Purpose" would be the description/purpose of the specific .c/.h/.rc etc. file.

Reply:
Thanks, new header committed